### ISSUE TITLE
Release Please must use chore to increment minor version. Today it is incrementing patch version

### ISSUE THREAD

**Issue by rafaelromcar-parabol** [ParabolInc/parabol#9086]
​

**Comment by mattkrick** [comment]
i don't think we can configure that part,  `feat` bump minor, `fix` bumps patch. i don't see any other semver triggers.
The workaround is to write `Release-As: 2.1.0` in the commit

**Comment by rafaelromcar-parabol** [comment]
I would avoid adding the release-as to a particular merge and just doing a manual commit --allow-empty with Release-As if needed. That would require a manual check and a human decision. Not dramatic, but sadly we can't do much here 😿

**Comment by mattkrick** [comment]
at the end of the day the version doesn't really matter. it's a app, not a lib, so nothing depends on our app to follow a strict semvar. only other option i see is making every release a new major change & then using the patch number for hotfixes & then we ignore the minor version entirely. feels less fun, but it's an option

**Comment by mattkrick** [comment]
OR we write `feat(release): release v1.0.0`.

**Comment by Dschoordsch** [comment]
I didn't look too close, but I see 2 options:
* we configure `always-bump-minor` and ignore the patch version for the most part (unless there is an easy way to apply a different config for hotfix branches)
* it seems we could implement a custom [`VersioningStrategy`](https://github.com/googleapis/release-please/blob/6a1ccc9ba749e5c81c4d2db4fb54a1b27feed553/src/versioning-strategy.ts#L126)

**Comment by jordanh** [comment]
Came here to scrub this.

This feels like governance, not a backlog issue. @rafaelromcar-parabol what would you like to do here to close this issue?

**Comment by rafaelromcar-parabol** [comment]
For me it isn't a governance, as it is related with how Release Please is configured and its capabilities.

For me this is a very very very low prio task, but it is something we should do at some point. I would like to test and play with Release Please so the app version increments as minor when there are chores, instead of incrementing patch versions when there are chores.

**Comment by jordanh** [comment]
Scrubbed again. Adjusted labels.

We'll weigh the priority of this in subsequent cycles or somebody can tackle during cooldown. If it comes stale again, we'll close

**Comment by jordanh** [comment]
Iceboxing for now. @rafaelromcar-parabol please reopen if this is important to you.

**Comment by rafaelromcar-parabol** [comment]
Not really, not if that doesn't bother anyone
